.github/workflows/pytest.yml

```diff
@@ -1,13 +1,19 @@
-name: Pull Request
+name: Check & Test
 
 on:
+  push:
+    branches:
+      - develop
+      - master
+      - main
+      - beta
   pull_request:
     branches:
+      - master
       - develop
-      - django4
+      - beta
```

Comment:
I think, we should we add `release/*` here?

Reply:
Good catch. Most probably we should.